docs/development/setup-advanced.md

```diff
@@ -373,6 +374,34 @@ proxy in the environment as follows:
  yarn config set https-proxy http://proxy_host:port
  ```
 
+## Installing directly on cloud9
+This section documents how to setup zulip dev environment on cloud9 workspace.
+If you don't have a cloud9 account signup [here](https://aws.amazon.com/cloud9/)
+although new AWS cloud9 is not free. And the zulip dev env may work with zulip-cloud9 package.
+
+* Create a Workspace, and select the blank template.
+* Resize the workspace to be 1GB of memory and 4GB of disk space. (This is under free limit)
+* Clone the zulip repo `git clone --config pull.rebase https://github.com/<your-username>/zulip.git`
+* Restart rabbitmq-server since its broken on cloud9 `sudo service rabbitmq-server restart`.
+* And run provison `cd zulip && ./tools/provision`, once this is done.
+* Activate virtual environment by `source /srv/zulip-py3-venv/bin/activate` or by opening a new terminal.
+
+#### Install zulip-cloud9
+Note: `npm i -g zulip-cloud9` does not work in zulip's virtual enviorment.
```

Comment:
One can leave the virtualenv with `deactivate`; maybe helpful to mention?  Or we should make this work :)

Reply:
That would still not work since we use different node version in virtualenv. So deactivate will node work, I do think we should make it work in virtualenv but not priority.

Reply:
OK, can you make a follow-up issue for fixing this?